### ISSUE TITLE
Color issue introduced in commit 629e24

### ISSUE THREAD

**Issue by 7415963987456321** [jwilm/alacritty#2759]
Which operating system does the issue occur on?
>(linux 5.1.5)

If on linux, are you using X11 or Wayland?
> Wayland (sway)

While testing the latest master I ran into an issue where the entire terminal will turn green.
Doing a git bisect points towards commit 629ea247cdccc33c6df0037f357ba9be48c7178d
I looked over the commit but could not find any obvious issues with the code, but then again I barely know rust. 

Let me know I you need any more info in order to debug.

**Comment by kchibisov** [comment]
Would you mind creating ref test `alacritty --ref-test` in case you can reliably repro the issue?

P.s. Close your alacritty with WM hotkey and not `Ctrl-D` (due to this #2625)

**Comment by 7415963987456321** [comment]
Here's the output from --ref-test, is it supposed to be like this?
[alacritty.recording](https://gist.github.com/7415963987456321/818b56394fb1de8374e6f115c7d11609)

I thought for a second that the issue was with my greeting message, a small ascii logo and output from neofetch run through lolcat, however it seems that disabling that feature still results in everything being green.

**Comment by 7415963987456321** [comment]
Just realized that runnng alacritty with -e
`alacritty -e htop`

results in normal output.

**Comment by chrisduerr** [comment]
Do you have a script which tries to set your terminal colors outside from what is done through the Alacritty config?

Also, just to make clear, when you shut down the ref test, the entire terminal was green at that point, right? Ref tests replay all escapes that were sent to the application, so this is only useful if the last state your terminal was in was  erroneous.

**Comment by 7415963987456321** [comment]
@nixpulvis I usually run fish shell, setting the shell to bash(I have no custom options in my bash setup)  with chsh results in the shell being displayed normally. 
@chrisduerr Ah, yes of course, forgot to mention that I run pywal, that might be the culprit.  I just tried reloading another pywal colorscheme and that changes the result. So my pywal color-scheme is generated from my wallpaper, while rolling through some wallpapers I noticed that about half of them give the terminal the green color, sometimes varying in tint, while others work normally. So it seems the issue lies with how pywal is changing the colorscheme of the terminal.  I don't know why it has started doing the green background all of a sudden though... is this a bug with how alacritty interprets the colorscheme from pywal? Everything used to work fine before commit 629ea247cdccc33c6df0037f357ba9be48c7178d .
